modules/BuildTools.Tasks/BuildTools.Tasks.csproj

```diff
@@ -9,16 +9,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\shared\Microsoft.Extensions.CommandLineUtils.Sources\**\*.cs" />
```

Comment:
Not exactly. 3rd party deps are on NuGet.org, but Ms.Ext.CommandLineUtils.Sources won't be.